README.md

```diff
@@ -1,180 +1,143 @@
-# napi
+<p align="center">
+  <img src="https://raw.githubusercontent.com/nanoapi-io/napi/refs/heads/main/media/logo-192.png" alt="NanoAPI Logo" width="200" />
+</p>
 
-Powerful CLI + UI for inspecting and refactoring an API codebase in any language and web framework.
+# `napi` - Next-Level Visual Tooling For API Codebases
 
-> This project is under ongoing development. Check the [status section](#status) for more information on supported languages and frameworks.
+`napi` is a versatile tool designed to automatically refactor large microlith codebases into smaller, more manageable microservices. With both a powerful CLI and an intuitive UI, `napi` is compatible with all major CI/CD platforms, allowing seamless integration into your development and deployment pipelines.
 
-## Features
+![NanoAPI UI Screenshot](/media/screenshots/app.png)
```

Comment:
We're missing the screenshots/ dir for app.png and app-ui.png

@joeldevelops

Reply:
I'll work on adding some